tests/console/login.pm

```diff
@@ -0,0 +1,26 @@
+package login;
+
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: login console
+# Maintainer: Tony Yuan <tyuan@suse.com>, qe-virt@suse.com
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+#use utils;
+#use virt_autotest::utils;
+#use virt_autotest::common;
+use lib 'sle/tests/virt_autotest';
+use lib 'os-autoinst-distri-opensuse/tests/virt_autotest';
+
+sub run {
+    my $self = shift;
+    select_console 'root-ssh';
+    record_info("ok");
+}
```

Comment:
I think you can use following subroutine in lib/Utils/Backend.pm
```
sub use_ssh_serial_console {
    select_console('root-ssh');
    set_sshserial_dev;
}
```
Or you should also add set_sshserial_dev into your new login.pm ?

Reply:
I don't see that any of qam tests need use variable "SERIALDEV" so "set_sshserial_dev" is not necessary here.

Reply:
That's OK.